amazonlinux2/bin/sts-assume-role.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
These two scripts we had to copy in for now. The docker concourse resource seems to hard-code the directory that contains the `Dockerfile` as the context directory - nothing can be used outside of that. Maybe we could wget these two scripts in the future? Something to think about.